v-next/hardhat-verify/src/internal/hook-handlers/config.ts

```diff
@@ -62,13 +70,23 @@ export async function resolveUserConfig(
   };
 }
 
+function resolveBlockscoutConfig(
+  blockscoutConfig: BlockscoutUserConfig | undefined = {
+    enabled: false,
+  },
+): BlockscoutConfig {
+  return {
+    enabled: blockscoutConfig.enabled ?? true,
```

Comment:
Does this mean we always try to verify with blockscout even if no blockscout config is set by the user? Is that expected?